### DISCUSSION TITLE
Ridiculous sidebar resizing

### DISCUSSION THREAD

**Discussion by bergdesign** [ViennaRSS/vienna-rss#1862] [Q&A]
I'm curious if I'm the only one seeing the insane sidebar resizing when switching from an internal laptop display to an external display of a different resolution in clamshell mode. I'm typically in the horizontal layout, and I see the sidebar widen and narrow instead of the articles list and content view. To add, I typically have the window maximized (but not full screen) on the smaller internal display, and it automatically maximizes on the larger external display making the sidebar like five times wider than the content view. Long time Obj-C dev, so happy to help troubleshoot if this is something new... I was unable to find anything related in issues.

**Comment by barijaona** [comment]
I could not reproduce this, probably because the external displays I could test on (1920x1080) have lower resolution than the built-in screen.

**Reply by Eitot** [reply]
I was able to reproduce it on my MBP's internal display simply by entering full screen. I think the underlying cause may be the same, namely when the window's size changes while the size of the article view is fixed by Auto Layout constraints, the sidebar resizes. A similar thing happens with the vertical layout, although here it is the article list that changes its width, not the sidebar.

I opened a bug report #1863
